Makefile

```diff
@@ -31,7 +31,6 @@ export PATH := $(shell pwd)/toolchain:$(PATH)
 
 # Moveable Directories based on the caller makefile
 TOP_DIR ?= $(UT_CORE_DIR)
-BUILD_DIR ?= $(TOP_DIR)/obj
 BIN_DIR ?= $(TOP_DIR)/bin
 LIB_DIR ?= $(TOP_DIR)/lib
```

Comment:
our defaults all need to be 

LIB_DIR should be to /build/<target>/lib
BIN_DIR should be to /build/<target>/bin
OBJ_DIR should be to /build<target>/obj
BUILD_DIR should be $(TOP_DIR)/build/<target>/obj

and we'll have to fix the testing suites to reflect that also.

Reply:
This is being following in ut-core and ut-control:
LIB_DIR : /build/${TARGET}/lib
BUILD_DIR : build/${TARGET}/obj
BIN_DIR : /build/bin
OBJ_DIR : no directory like that currently